README.md

```diff
@@ -14,23 +17,46 @@
 
 [Task 5](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task05.ipynb): Post-processing and visualisation
 
-[Task 6](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task06.ipynb): Advection-diffusion problem
-
 [Task 7](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task07.ipynb): Heat transfer simulation
 
 [Task 8](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task08.ipynb): CAD integration
 
 [Task 9](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task09.ipynb): Integration with the HTM library
 
+
+### Advanced
+
+[Task 6](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task06.ipynb): Advection-diffusion problem
+
 [Task 10](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task10.ipynb): Fitting a TDS spectrum
 
+[Task 11](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task11.ipynb): Radioactive decay ☢️
+
+[Task 12](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task12.ipynb): Soret Effect
+
+[Task 13](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task13.ipynb): Modelling discontinuous trapped concentration profiles
+
+[Task 14](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task14.ipynb): Non-cartesian meshes
+
+[Task 15](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task15.ipynb): Kinetic surface model
+
+[Task 16](https://github.com/festim-dev/FESTIM-workshop/blob/main/tasks/task16.ipynb): Custom classes
+
+
+
 ## Getting started
 
 ### A. Codespaces (recommended)
 
 You can [create a Codespace](https://github.com/codespaces/new?machine=standardLinux32gb&repo=520445592&ref=main&devcontainer_path=.devcontainer%2Fdevcontainer.json&location=WestEurope) based on this repo
 
-### B. Local install
+### B. Binder

```

Comment:
Also could have Binder as the primary choice given the reliability issues with codespaces